docker/prod/prod.template.yaml

```diff
@@ -370,8 +400,29 @@ spec:
       type: mediawiki
   namespaceSelector:
     matchNames:
-    - prod
+      - prod
   endpoints:
-  - port: main
-    path: /metrics.php
-    interval: 10s
+    - port: main
+      path: /metrics.php
+      interval: 10s
+---
+apiVersion: monitoring.coreos.com/v1
```

Comment:
There is already one `ServiceMonitor` defined above - you should probably add a new entry there instead of creating a new one